selfdrive/ui/translations/README.md

```diff
@@ -31,6 +31,14 @@ openpilot provides a few tools to help contributors manage their translations an
    ```shell
    scons -j$(nproc) selfdrive/ui && selfdrive/ui/ui
    ```
+5. Trigger offRoad alerts (clear OffRoad params later):
+   ```shell
+   selfdrive/ui/tests/cycle_offroad_alerts.py
+   ```
+6. Trigger onRoad alerts (hardcode with the alerts you need to see):
+   ```shell
+   selfdrive/debug/cycle_alerts.py
+   ```
```

Comment:
Can you make this a separate section about viewing translations in the UI?

Reply:
![image](https://github.com/commaai/openpilot/assets/66435071/5ce1cd50-7974-417a-9a18-7f5ee712974f)